<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MercurialPage</class>
 <widget class="QWidget" name="MercurialPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>384</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Mercurial Interface&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line15">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>No. of log messages shown:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="logSpinBox">
        <property name="toolTip">
         <string>Enter the number of log messages to be shown</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Commit</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>No. of commit messages to remember:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="commitSpinBox">
        <property name="toolTip">
         <string>Enter the number of commit messages to remember</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Incoming / Outgoing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="logBrowserCheckBox">
        <property name="toolTip">
         <string>Select to show the incoming and outgoing log in a log browser dialog</string>
        </property>
        <property name="text">
         <string>Use Log Browser for incoming / outgoing log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Pull</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="pullUpdateCheckBox">
        <property name="toolTip">
         <string>Select to update the working directory to new tip</string>
        </property>
        <property name="text">
         <string>Update after pulling</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="configButton">
     <property name="toolTip">
      <string>Edit the Mercurial configuration file</string>
     </property>
     <property name="text">
      <string>Edit configuration file</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>388</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>logSpinBox</tabstop>
  <tabstop>commitSpinBox</tabstop>
  <tabstop>logBrowserCheckBox</tabstop>
  <tabstop>pullUpdateCheckBox</tabstop>
  <tabstop>configButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
